marketplace_confg/schema.yaml

```diff
@@ -27,129 +27,162 @@ properties:
+      (Optional) Transfer Syntax to convert instances to during C-STORE 
+      upload. See Readme for list of supported syntaxes.
+  import.monitoring_project_id:
+      type: string
+      title: "importer - Monitoring project id (Optional)"
+      description: >-
+        (Optional) Stackdriver monitoring project id, must be the same as the 
+        project id in which the adapter is running.
+  import.custom_flags:
+      type: string
+      title: "Extra import adapter flag (Optional)"
+      description: >-
+        (Optional) Addtional flag for import adapter.
+        e.g. "--flag_a=valueA"
 
-  export.hasexporter:
-    type: boolean
-    title: Enable dicom exporter
-    description: Set to true if you want to add a dicom exporter to the deployment.
-    default: false
-    enum:
-    - true
-    - false
   export.peer_dimse_aet:
     type: string
-    title: "exporter - peer DIMSE aet"
-    description: "(CStore) Application Entity Title of DIMSE peer."
+    title: "exporter - Peer DIMSE aet"
+    description: (CStore) Application Entity Title of DIMSE peer.
   export.peer_dimse_ip:
     type: string
-    title: "exporter - peer DIMSE IP"
-    description: "(CStore) IP of DIMSE peer."
+    title: "exporter - Peer DIMSE IP"
+    description: (CStore) IP of DIMSE peer.
   export.peer_dimse_port:
     type: integer
-    title: "exporter - peer DIMSE port"
-    description: "(CStore) Port of DIMSE peer."
-  export.peer_dicomweb_address:
-    type: string
-    title: "exporter - peer dicomweb address"
-    description:  >-
-      "(StowRs) Address of peer DicomWeb API serving STOW-RS. Must be a full path up 
-      to /dicomWeb if the Cloud Healthcare API is used."
-  export.use_gcp_application_default_credentials:
-    type: boolean
-    title: "exporter - Enable gcp application default credentials"
-    description:  >-
-      "(StowRs) If true, uses GCP Application Credentials to when sending HTTP requests 
-      to peer. This is useful if the peer DICOMWeb endpoint is running in GCP."
-    enum:
-    - true
-    - false
+    title: "exporter - Peer DIMSE port"
+    description: (CStore) Port of DIMSE peer.
   export.project_id:
     type: string
-    title: "exporter - pubsub project id"
+    title: "exporter - Pubsub project id"
     description:  >-
       Pubsub project id, must be the same as the project id in which the adapter is running.
   export.subscription_id:
     type: string
-    title: "exporter - pubsub subscription id"
+    title: "exporter - Pubsub subscription id"
     description: The pubsub subscription id
   export.dicomweb_addr:
     type: string
-    title: "exporter - dicom web address"
-    description: Address for DicomWeb service that sources the DICOM.
+    title: "exporter - Dicom web address"
+    description: >-
+      Address for DicomWeb service that sources the DICOM. e.g. 
+      https://healthcare.googleapis.com/v1/projects/myproject/locations/
```

Comment:
It should actually just be this:
"https://healthcare.googleapis.com/v1"
not a full path.

Reply:
Changed